src/containers/user-profile/profile-info/ProfileInfo.tsx

```diff
@@ -132,6 +139,31 @@ const ProfileInfo = ({ userData, myRole }: ProfileInfoProps) => {
       }
   ].filter(Boolean)
 
+  const {
+    response: listOfChats,
+    loading: isChatsLoading,
+    fetchData
+  } = useAxios({
```

Comment:
Add generic type to the `useAxios` hook, and you will not have that weird types assertions in finding existing chat, because right now `listOfChats` has `never` type
```suggestion
  } = useAxios<ChatResponse[]>({
```